static/tinymce_templates/parking.html

```diff
@@ -0,0 +1,33 @@
+<b>Parkordnung</b>
+<table class="table table-striped table-sm">
```

Comment:
Why is this file located in this directory?

Reply:
I did not read up on tinymce templates yet, but it seems to make sense to have templates here if they exist...